### PR TITLE
Dining reorder

### LINKED ISSUE #290: Dining menus out of order

### PR DESCRIPTION
**Pull request by dlakata**
Fixes #290 and removes NewDiningHall

### CODE REVIEW

**Comment by dlakata** [comment]
@jasontsh @lilyidi

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/DiningAdapter.java [hunk 1]
```diff
@@ -75,28 +74,28 @@ public void call(Throwable throwable) {
         }
 
         if (diningHall.isOpen()) {
-            holder.hallStatus.setText("Open");
+            holder.hallStatus.setText(R.string.dining_hall_open);
             holder.hallStatus.setBackground(ContextCompat.getDrawable(getContext(), R.drawable.label_green));
             if (!diningHall.openMeal().equals("all")) {
-                holder.openMeal.setText("Currently serving " + diningHall.openMeal());
+                holder.openMeal.setText(String.format("Currently serving %s", diningHall.openMeal()));
```
Comment (adelq) [inline PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/DiningAdapter.java]: Is C finally growing on you?

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/DiningHall.java [hunk 2]
```diff
@@ -92,18 +97,15 @@ public boolean hasMenu() {
     }
 
     public String closingTime() {
-        String closingTime = "";
         for (Interval openInterval : openHours.values()) {
-            DateTime currentTime = new DateTime();
-            if (openInterval.contains(currentTime)) {
-                closingTime = openInterval.getEnd().toString("h:mma");
-                return closingTime;
+            if (openInterval.containsNow()) {
```
Comment (adelq) [inline PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/DiningHall.java]: the more you know

**Comment by adelq** [comment]
Legendary pull request, LGTM

**Comment by jasontsh** [comment]
No more new dining halls! LGTM

**Pull request merged**